homeassistant/helpers/intent.py

```diff
@@ -253,19 +269,40 @@ def __init__(self, intent: Intent | None = None) -> None:
         self.reprompt: dict[str, dict[str, Any]] = {}
         self.card: dict[str, dict[str, str]] = {}
 
+    @property
+    def language(self) -> str | None:
+        """Return intent language if available."""
+        return self.intent.language if self.intent is not None else None
```

Comment:
If it’s none you’re returning none so no need for if check

Reply:
Setting this in the constructor now